classes/Task/Upgrade/BackupDb.php

```diff
@@ -215,49 +162,71 @@ public function run(): int
             }
             // end of schema
 
+            $i = 0;
+
             // POPULATE TABLE
-            if (!in_array($table, $ignore_stats_table)) {
-                do {
-                    $backup_loop_limit = $this->container->getState()->getBackupLoopLimit();
-                    $data = $this->container->getDb()->executeS('SELECT * FROM `' . $table . '` LIMIT ' . (int) $backup_loop_limit . ',200', false, false);
-                    $this->container->getState()->setBackupLoopLimit($this->container->getState()->getBackupLoopLimit() + 200);
-                    $sizeof = $this->container->getDb()->numRows();
-                    if ($data && ($sizeof > 0)) {
-                        // Export the table data
+            if ($this->container->getState()->getBackupTable()) {
+                $backup_loop_limit = $this->container->getState()->getBackupLoopLimit();
+
+                $dbLink->setAttribute(PDO::MYSQL_ATTR_USE_BUFFERED_QUERY, false);
+                /** @see https://dev.mysql.com/doc/refman/8.4/en/select.html specifies a large LIMIT value to get the whole table */
+                $data = $dbLink->prepare('SELECT * FROM `' . $table . '` LIMIT ' . (int) $backup_loop_limit . ',18446744073709551615');
```

Comment:
it might be worth outputting 18446744073709551615 as a constant, with a variable name that could help us understand the number

Reply:
I though about it and preferred to put it here as 
* this is a value we won't change in the future, and
* it is the closest possible to its comment and the context in which it is used (the SQL request).